api_v2/src/leaves/leaves.resolver.ts

```diff
@@ -45,7 +49,13 @@ export class LeavesResolver {
 
   @Mutation(() => [LeaveDTO])
   async createLeave(@Args('leave') leave: CreateLeaveRequestInput) {
-    return await this.leavesService.createLeave(leave);
+    const leaves = await this.leavesService.createLeave(leave);
+    for (const leave of leaves) {
```

Comment:
Maybe we can remove the console.log here and replace this with
```suggestion
    await Promise.all(
    leaves.map((leave) =>
      this.notificationService.createLeaveNotification(leave)
    )
  );
```